playwright.config.ts

```diff
@@ -0,0 +1,41 @@
+import { defineConfig, devices } from "@playwright/test";
+import {
+  createReplayReporterConfig,
+  devices as replayDevices,
+} from "@replayio/playwright";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+// This is a temporary workaround so that replays in CI can be public by default.
+// Should be fixed soon.
+const replayReporterOptions = process.env.CI
+  ? {}
+  : {
+      upload: true,
+      apiKey: process.env.REPLAY_API_KEY,
+    };
```

Comment:
```suggestion
const replayReporterOptions = {
  upload: process.env.CI,
  apiKey: process.env.REPLAY_API_KEY,
};
```

Reply:
This is needed so that in CI replays are public by default

Reply:
Ooops I inverted the CI conditional bc I forgot we needed to use the upload action. Could still use an inline though with `!process.env.CI`)